extensions/Prefixes.md

```diff
@@ -36,6 +36,7 @@ Once a prefix is approved, it should be added to the [glTF Validator's list of k
 | `CITRUS` | Citrus Toolbox | https://github.com/KenzieMac130/CitrusToolbox | [#1962](https://github.com/KhronosGroup/glTF/issues/1962) |
 | `CLO` | CLO Virtual Fashion | https://www.clo3d.com/ | [#1944](https://github.com/KhronosGroup/glTF/issues/1944) |
 | `CVTOOLS` | Canvas Tools | https://github.com/MackeyK24/CanvasTools | [#1389](https://github.com/KhronosGroup/glTF/issues/1389) |
+| `DGG` | Darmstadt Graphics Group GmbH | https://dgg3d.com/ | [#2303](https://github.com/KhronosGroup/glTF/issues/2303) |
```

Comment:
This link responds with HTTP 301 Moved Permanently to `https://rapidcompact.com/about/`.

@mlimper please confirm the URL to use here.

Reply:
Similar to https://github.com/KhronosGroup/glTF/pull/2265#discussion_r1133274626 

(Even though this may be a case where the 301 is used _intentionally_ as some sort of "URL-shortener"...)